packages/core/admin/admin/src/layouts/UnauthenticatedLayout.tsx

```diff
@@ -11,12 +11,14 @@ import {
 import { useIntl } from 'react-intl';
 import { styled } from 'styled-components';
 
+import { RESPONSIVE_DEFAULT_SPACING } from '../constants/theme';
 import { useTypedDispatch, useTypedSelector } from '../core/store/hooks';
 import { setLocale } from '../reducer';
 
 const Wrapper = styled<BoxComponent>(Box)`
   margin: 0 auto;
-  width: 552px;
+  width: 100%;
+  max-width: 552px;
```

Comment:
any reason to not use rem unit here?
```suggestion
  max-width: 55.2rem;
```

Reply:
It was already like that 🤷‍♂️ But you are right, I'll make the change